### ISSUE TITLE
Invert Icon

### ISSUE THREAD

**Issue by devNoiseConsulting** [rubencodes/Flat-Icon-Generator#2]
Is it possible to invert the color of the icon? Depending on the color you choose for the background, the icon may look better as white than black.

**Comment by rubencodes** [comment]
Agreed! More generally I think it would be useful to be able to set the hex value for the color. I'll put this on my to-do list.
